### PR TITLE
feat: Update development container with elastic stack 7.10 version 

### PR DESCRIPTION
**Pull request by dupuyjs**
# PR Request

This container is based on image `mcr.microsoft.com/vscode/devcontainers/base:0-ubuntu20.04` which already contains a set of dev tools and linux packages.

The following packages are installed on top of this base image.

| Tool / library | Version |
|----------------|---------|
| terraform | 1.0.9 |
| azure-cli | latest |
| helm | latest |
| mitmproxy | 7.0.4 |
| dotnet-sdk-6.0 | latest |
| dotnet-sdk-5.0 | latest |
| default-jre | latest |
| elasticsearch-oss| 7.10.2 |
| kibana-oss | 7.10.2 |

Added a change to devcontainer.json to mount local user ./ssh folder. (used for github authentication with ssh).

### CODE REVIEW

**Review by tomconte — APPROVED** [review]
Looks good 👍🏻

**Review by eladiw — APPROVED** [review]

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Install Helm
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+RUN chmod 700 get_helm.sh
+RUN ./get_helm.sh
+
+# Install mitmproxy
+ENV MITMPROXY_VERSION=${BUILD_MITMPROXY_VERSION}
+RUN wget https://snapshots.mitmproxy.org/${MITMPROXY_VERSION}/mitmproxy-${MITMPROXY_VERSION}-linux.tar.gz
+RUN tar -xzf mitmproxy-${MITMPROXY_VERSION}-linux.tar.gz -C /usr/local/bin
+RUN rm mitmproxy-${MITMPROXY_VERSION}-linux.tar.gz
```
Comment (eladiw) [inline .devcontainer/Dockerfile:28]: why do we need this?

two options as I see it:
a) leave it and provide info how to use it in the dev container docs
b) remove it

**Review by eladiw — CHANGES_REQUESTED** [review]
One question regarding mitmproxy

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (dupuyjs) [reply .devcontainer/Dockerfile:28]: OK, will work on option a).

Comment (dupuyjs) [reply .devcontainer/Dockerfile:28]: @eladiw @tomconte Just added documentation page for mitmproxy. In addition, I removed dotnet-sdk-3.1 and added dotnet-sdk-6.0 instead (so this base image contains dotnet-sdk 5.0 and 6.0).

**Review by eladiw — APPROVED** [review]

**Pull request merged**